.gitignore

```diff
@@ -65,3 +65,10 @@ test/mismatch/
 test/outputs/
 win32/BUGS.txt
 win32/COPYING.txt
+
+# Visual Studio code files
+.vscode/
+
+# files generated by running test/run_tests.sh
+test/inputs/test-image-rotation-1-rotated.gbx
```

Comment:
Thanks for this, it had always been so annoying. It would be nicer if those temporary files were built into an outputs directory and e could ignore that whole directory. But this is fine, too.

If one day we have lots of these then we'll rethink it.